Latest updates (minor) (#4546)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/install-app-guard.md

### Minimum software requirements

* **Windows 10**: Windows 10 Enterprise edition, Client Build version 2004 (20H1) build 19041 or later
* **Office**: Office Current Channel and Monthly Enterprise Channel, Build version 2011 16.0.13530.10000 or later. Both 32-bit and 64-bit versions of Office are supported.
* **Update package**: Windows 10 cumulative monthly security update [KB4571756](https://support.microsoft.com/help/4571756/windows-10-update-KB4571756)

> For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). Please refer to your computer manufacturer guides on how to enable virtualization technology.

Reviewer: ```suggestion
For detailed system requirements, refer to [System requirements for Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/reqs-md-app-guard). Also, please refer to your computer manufacturer's guides on how to enable virtualization technology.
```

## Configure Application Guard for Office

> Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service](/DeployOffice/overview-office-cloud-policy-service).

Reviewer: ```suggestion
Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office cloud policy service](/DeployOffice/overview-office-cloud-policy-service).
```

> Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service] (/DeployOffice/overview-office-cloud-policy-service).

Reviewer 2: double-check in preview if the space between ] and ( affects render of hyperlink

> Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service] (/DeployOffice/overview-office-cloud-policy-service).

Reviewer: ```suggestion
Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the [Office Cloud Policy Service](/DeployOffice/overview-office-cloud-policy-service).
```

> See configuration set by your administrator by reviewing group policy settings in  **User Configuration\\Administrative Templates\\Microsoft Office 2016\\Security Settings\\ Trust Center\\Application Guard**.

Reviewer 2: remove space before Trust

> See configuration set by your administrator by reviewing group policy settings in  **User Configuration\\Administrative Templates\\Microsoft Office 2016\\Security Settings\\ Trust Center\\Application Guard**.

Reviewer: ```suggestion
See configuration set by your administrator by reviewing group policy settings in  **User Configuration\\Administrative Templates\\Microsoft Office 2016\\Security Settings\\Trust Center\\Application Guard**.
```

Application Guard for Office is integrated with Microsoft Defender for Endpoint to provide monitoring and alerting on malicious activity that happens in the isolated environment.

> [Safe Documents in Microsoft E365 E5] (/microsoft-365/security/office-365-security/safe-docs) is a feature that uses Microsoft Defender for Endpoint to scan documents opened in Application Guard for Office. For an additional layer of protection, users can't leave Applicatio Guard for Office until results of the scan have been determined.

Reviewer: ```suggestion
[Safe Documents in Microsoft E365 E5](/microsoft-365/security/office-365-security/safe-docs) is a feature that uses Microsoft Defender for Endpoint to scan documents opened in Application Guard for Office. For an additional layer of protection, users can't leave Application Guard for Office until the results of the scan have been determined.
```

* Selecting web links (`http` or `https`) doesn't open the browser.
* Pasting rich text format (RTF) content or images in Office documents opened with Application Guard isn't supported at this time.

> * Default setting for unsuported file types protection policy is to block opening untrusted unsupported file types of IRM (Information Rights Management), CSV or HTML

Reviewer 2: edit to:

Default setting for unsupported file types protection policy is to block opening untrusted and/or unsupported file types including IRM (Information Rights Management), CSV or HTML.

> * Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).

Reviewer 2: edit to:

Updates to .NET can cause files to fail to open in Application Guard.

also, can't verify if there is a space between ] and ( for the hyperlink. if there is, kindly remove.

* Selecting web links (`http` or `https`) doesn't open the browser.
* Pasting rich text format (RTF) content or images in Office documents opened with Application Guard isn't supported at this time.

> * Default setting for unsuported file types protection policy is to block opening untrusted unsupported file types of IRM (Information Rights Management), CSV or HTML

Reviewer: ```suggestion
* The default setting for unsupported file types protection policy is to block opening untrusted unsupported file types of Information Rights Management (IRM), CSV, or HTML.
```

Author: There isn't a condition for "and/or". I like the updates but without the and/or. 
"The default setting for unsupported file types protection policy is to block opening untrusted unsupported file types of Information Rights Management (IRM), CSV, or HTML"

Reviewer: Thank you for your feedback. I have edited my suggestion accordingly. Could you please commit this final suggestion? Many thanks.

> * Updates to .NET cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).

Reviewer: ```suggestion
* Updates to .NET might cause files to fail to open in Application Guard. As a workaround, users can restart their device when they come across this failure. Learn more about the issue at [Receiving an error message when attempting to open Windows Defender Application Guard or Windows Sandbox](https://support.microsoft.com/help/4575917/receiving-an-error-message-when-attempting-to-open-windows-defender-ap).
```
